app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class PostsController < ApplicationController
+  def index
+    @post = Post.new
+    @user_posts = Post.all #current_user.posts
+  end
+
+  def create
+    @user = current_user
```

Comment:
- [ ] **[OPTIONAL]** `current_user` is already available in our views. We can remove this line and use `current_user` for the post creation.

Reply:
removed line 10 and set @post = current_user.posts.build(post_params) as required by reviewer.